### PR TITLE
vendor: switch to govmm

### LINKED ISSUE #519: switch to govmm
replace ciao-project with govmm

### PR DESCRIPTION
**Pull request by devimc**
This patch replaces ciao-project with govmm

github.com/intel/govmm/ was branched from
github.com/ciao-project/ciao at commit 239f5bc3e8e5fe18e6b4df31dcd692db5692c55f.

shortlog:
f565536 vhost-user: add blk device support
e9e2767 vhost-user: updating comments for accuracy, rename device field
8fe5723 qemu: Add maxcpus attribute to -smp
3baa776 Add badges to the README.md file
d74e3b6 Fix errcheck failures in the unit tests
db60e32 Enable Travis builds
9cb47fc Add .gitignore file.
a8aaf53 Add project documentation
57aafb5 Remove all references to and dependencies on ciao
27709fc Move files to the qemu folder

fixes #519

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Could you add details to the commit about this transition? FWICS, https://github.com/intel/govmm/ was branched from https://github.com/ciao-project/ciao at commit 239f5bc3e8e5fe18e6b4df31dcd692db5692c55f.

In other words I think, https://github.com/ciao-project/ciao/commit/239f5bc3e8e5fe18e6b4df31dcd692db5692c55f is the branch point and that commit became https://github.com/intel/govmm/commit/367ac50fe8fb55163369b6acf0a7028b452b17c5 in the new project but we might need confirmation from @markdryan on that.

**Comment by sboeuf** [comment]
@mcastelino well apparently the CI passed properly on that one.
LGTM

**Comment by mcastelino** [comment]
@sboeuf I wonder how CI passed for this. when we have not switched the runtime to use this package. 
This begs the question what is actually happening when we run the CI with runtime here.

File: Gopkg.toml [hunk 1]
```diff
@@ -132,7 +132,6 @@
   name = "github.com/sirupsen/logrus"
   revision = "89742aefa4b206dcf400792f3bd35b542998eb3b"
 
-
 [[constraint]]
-  name = "github.com/ciao-project/ciao"
-  revision = "239f5bc3e8e5fe18e6b4df31dcd692db5692c55f"
+  branch = "master"
```
Comment (mcastelino) [inline Gopkg.toml]: Why did we switch the constraint to be just master?

Comment (sboeuf) [reply Gopkg.toml]: Oh yeah I didn't catch that, @devimc we want to stick with a specific revision of govmm here.

Comment (devimc) [reply Gopkg.toml]: ooops, fixed

**Comment by devimc** [comment]
Ready to merge :)

**Pull request merged**